dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facilities.repo.ts

```diff
@@ -145,42 +145,43 @@ export class TfmFacilitiesRepo {
    * @param status - The amendment status
    * @returns The found amendments
    */
-  public static async findAmendmentsByFacilityIdAndStatus(facilityId: string | ObjectId, status: AmendmentStatus): Promise<TfmFacilityAmendment[]> {
+  public static async findAmendmentsByFacilityIdAndStatus(facilityId: string | ObjectId, status: AmendmentStatus): Promise<FacilityAmendment[]> {
```

Comment:
It feels odd that in "TFM facilities repo - find one amendment ...", we use `FacilityAmendment` type instead of `TfmFacilityAmendment`, and `TfmFacilityAmendment` still exists?

Reply:
We decided it would be best to store the portal amendments in the same place as TFM facilities (since any amalgamation into `current values` needs to include amendments from portal as well as TFM). This does mean some of these methods & endpoints will need updating, which I/Beth will do once the new endpoints are included.

I'd be happy to chat more about this because I do appreciate its a bit weird but I think makes the most sense, rather than risking creating a second source of truth in portal facilities